package.json

```diff
@@ -19,7 +19,7 @@
     "@types/lodash": "^4.14.85",
     "@types/ws": "^3.2.0",
     "bignumber.js": "^4.1.0",
-    "https-proxy-agent": "^1.0.0",
+    "https-proxy-agent": "^2.0.0",
```

Comment:
Looks like the vuln was fixed in 2.2.0. Can we make this "2.2.1" (no `^`) to make sure we get the version we expect?

Due to https://github.com/ripple/ripple-lib/pull/882 we'd like to use specific versions to reduce unpredictability.

Reply:
Sure, done